src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -192,6 +192,7 @@ namespace OutputReportTabular {
 	int UnitConvSize( 0 );
 
 	bool WriteTabularFiles( false );
+	bool AllocateLoadComponentArraysDoAllocate( true );
```

Comment:
@JasonGlazer Why did you pull this out of the anonymous namespace and extern it? I mean I see in the unit test you set this to true at the beginning. Why not use the EnergyPlusFixture which will set this to true via clear_state. Then you do not need to extern AllocateLoadComponentArraysDoAllocate.